README.md

```diff
@@ -27,10 +27,10 @@ Settings view (<kbd>cmd-,</kbd>).
 
 You can customize matching pairs in Bracket Matcher at any time. You can do so either Globally via the Settings view (<kbd>cmd-,</kbd>) or at the Scope level via `config.cson` (<kbd>cmd-shift-p</kbd> + "config"). Changes take effect immediately.
 
-* <b>Autocomplete Characters</b> - Comma-separated pairs that the editor will treat as brackets / quotes. Entries in this field override the package defaults.
+* **Autocomplete Characters** - Comma-separated pairs that the editor will treat as brackets / quotes. Entries in this field override the package defaults.
   * ie: `<>, (), []`
```

Comment:
Try to avoid latin abbreviations: use "for example" instead.
(See https://github.com/atom/flight-manual.atom.io/blob/master/CONTRIBUTING.md#language)